Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -436,7 +436,7 @@ extension FirefoxHomeViewController: UICollectionViewDelegateFlowLayout {
         case .topSites:
             return Section(section).footerHeight
         case .libraryShortcuts:
-            return UIDevice.current.userInterfaceIdiom == .pad ? CGSize.zero : Section(section).footerHeight
+            return UIDevice.current.isPad ? CGSize.zero : Section(section).footerHeight
```

Comment:
I suggest replacing this with a UITraitCollection based approach. You could be on an iPad, but in split screen mode.